core/templates/services/rte-helper-modal.controller.ts

```diff
@@ -222,6 +225,11 @@ export class RteHelperModalComponent {
 
     this.customizationArgsForm = this.fb.group(formGroupControls);
 
+    this.rteFieldValues = this.customizationArgsForm.valueChanges.subscribe(
```

Comment:
This reads a bit weirdly. How do we name other subscriptions in the codebase? Let's follow the existing convention for doing that.

(The current name suggests that the variable stores the RTE field values, which I don't think it does.)

Reply:
Looked at other subscriptions in the codebase and all of them have "subscription" in the name. Would renaming into something like customizationArgsFormSubscription be better?